lib/hexpm_web/views/view_helpers.ex

```diff
@@ -301,6 +305,19 @@ defmodule HexpmWeb.ViewHelpers do
   defp pretty_month(11), do: "November"
   defp pretty_month(12), do: "December"
 
+  defp pretty_month_short(1), do: "Jan"
+  defp pretty_month_short(2), do: "Feb"
+  defp pretty_month_short(3), do: "Mar"
+  defp pretty_month_short(4), do: "Apr"
+  defp pretty_month_short(5), do: "May"
+  defp pretty_month_short(6), do: "June"
```

Comment:
nitpick: I don't have a strong opinion about it, but perhaps worth keeping it always to 3 characters and follow strftime `%b`:

```suggestion
  defp pretty_month_short(6), do: "Jun"
```